api/notifications.js

```diff
@@ -214,20 +213,37 @@ function getNotificationHelper(api, account, identifier, topCallback) {
 
   // Parse the Notification object from the notificationDetails
   function parseNotificationDetails(notificationDetails, callback) {
-    callback(null, NotificationParser.parse(notificationDetails));
+    callback(null, NotificationParser.parse(notificationDetails, urlBase));
+  }
+
+  function formatNotificationResponse(notificationDetails, callback) {
+    var responseBody = {
+      notification: notificationDetails
+    };
+
+    // Move client_resource_id to response body instead of inside
+    // the Notification
+    var client_resource_id = responseBody.notification.client_resource_id;
+    delete responseBody.notification.client_resource_id;
```

Comment:
I see that, but that would still be moving as the comment above says. Was client_resource_id in the notification response before? I don't see it in the example on the dev portal.